src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy

```diff
@@ -643,9 +643,18 @@ class PersonsResource extends Resource {
                 addBadRequest("${job.positionNumber} $notValidErrorPhrase position number " +
                         "for the given begin date.")
             }
-            if (employmentType == studentEmploymentType && !job.isValidStudentPositionNumber()) {
-                addBadRequest("Student position numbers must begin with one of these prefixes: " +
-                        "${JobObject.validStudentPositionNumberPrefixes.join(", ")}")
+            if (employmentType == studentEmploymentType) {
+                if (['TERME', 'TERMJ'].contains(job.changeReasonCode)
```

Comment:
Could make this a variable and reuse it
`bool termination = ['TERME', 'TERMJ'].contains(job.changeReasonCode)`

Reply:
Good point, fixed.